conf/machine/raspberrypi0-2w-64.conf

```diff
@@ -0,0 +1,17 @@
+#@TYPE: Machine
+#@NAME: RaspberryPi0 2 Wifi Development Board
+#@DESCRIPTION: Machine configuration for the RaspberryPi0 2 Wifi in 64 bits mode
+
+include conf/machine/raspberrypi3-64.conf
+
+MACHINEOVERRIDES := "${@'${MACHINEOVERRIDES}'.replace(':${MACHINE}',':raspberrypi3-64:${MACHINE}')}"
```

Comment:
I find this replace just superfluous. Isn't `MACHINEOVERRIDES =. "raspberrypi3-64:"` enough?

Reply:
No. It won't work because the order will be wrong.

Reply:
Let's keep it as such for now.